src/Nethermind/Nethermind.Monitoring/Config/MetricsConfig.cs

```diff
@@ -18,9 +18,10 @@ namespace Nethermind.Monitoring.Config
 {
     public class MetricsConfig : IMetricsConfig
     {
-        public bool Enabled { get; set; } = false;
-        public string PushGatewayUrl {get; set; } =  "http://localhost:9091/metrics";
-        public int IntervalSeconds {get; set; } = 5;
+        public int? ExposePort { get; set; } = null;
+        public bool PushEnabled { get; set; } = false;
```

Comment:
unfortunately not possible to change it without backwards compatibility redirections of settings